CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ listed in the changelog.
 ### Changed
 
 - Prevent existing image streams from being cleaned up if they are renamed in future versions ([#366](https://github.com/opendevstack/ods-pipeline/issues/366))
+- Use `NEXUS_*` envs when testing with the `gradle-sample-app` (https://github.com/opendevstack/ods-pipeline/pull/392)
```

Comment:
To keep the changelog short, we do not include doc/test only changes (see comment at the top). You can then apply the `skip changelog` label to the PR. Sorry for the nit pick ...